docs/builder.rst

```diff
@@ -177,6 +177,17 @@ If building the image completes successfully, then use this command to run the i
    /docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
    ...
 
+Alternatively, you can run the Attack Flow Builder with Docker Compose:
+
+* `install Docker <https://docs.docker.com/engine/install>`_
+* grab this repository source code
+* run the following command in a terminal:
```

Comment:
Rather than clone the repo, please add instructions to curl the docker-compose.yml and run that instead.

Reply:
done